packages/shared/src/hooks/useSquadChecklist.tsx

```diff
@@ -69,16 +70,20 @@ const useSquadChecklist = ({
         },
         actions,
       }),
-      [ActionType.SquadFirstPost]: createChecklistStep({
-        type: ActionType.SquadFirstPost,
-        step: {
-          title: 'Share your first post',
-          description:
-            'Share your first post to help other squad members discover content you found interesting.',
-          component: SharePostChecklistStep,
-        },
-        actions,
-      }),
+      [ActionType.SquadFirstPost]:
+        verifyPermission(squad, SourcePermissions.Post) &&
+        createChecklistStep({
+          type: ActionType.SquadFirstPost,
+          step: {
+            title: 'Share your first post',
+            description:
```

Comment:
Just note that this description is different depending on whether someone is new as well.